src/main/java/org/jenkinsci/plugins/workflow/steps/SynchronousNonBlockingStepExecution.java

```diff
@@ -72,7 +74,14 @@ public void stop(Throwable cause) throws Exception {
 
     @Override
     public void onResume() {
-        getContext().onFailure(new SynchronousResumeNotSupportedException());
+        StepContext context = getContext();
+        String stepFunctionDisplayName = null;
+        try {
+            stepFunctionDisplayName = context.getStepDisplayFunctionName();
+        } catch (IOException x) {
+            // ignore
```

Comment:
Log fine at least? Not clear when that exception would even be thrown here.

Reply:
right, fixed now. ty